tests/test_job.py

```diff
@@ -115,7 +115,7 @@ def test_ensure_mergeable_mr_not_approved(self):
             merge_job.ensure_mergeable_mr(merge_request)
 
         merge_request.fetch_approvals.assert_called_once()
-        assert 'Insufficient approvals' in str(exc_info.value)
+        assert 'Insufficient approvals (need ' in str(exc_info.value)
```

Comment:
Unfortunately hard/impossible to validate the number displayed there because of magic mocking, but presuming it should work:

```
        merge_request.fetch_approvals.assert_called_once()
>       assert 'Insufficient approvals (need 1 more)' in str(exc_info.value)
E       assert 'Insufficient approvals (need 1 more)' in "Insufficient approvals (need <MagicMock name='mock.fetch_approvals().approvals_required.__sub__()' id='140291451716896'> more)"
E        +  where "Insufficient approvals (need <MagicMock name='mock.fetch_approvals().approvals_required.__sub__()' id='140291451716896'> more)" = str(CannotMerge("Insufficient approvals (need <MagicMock name='mock.fetch_approvals().approvals_required.__sub__()' id='140291451716896'> more)"))
E        +    where CannotMerge("Insufficient approvals (need <MagicMock name='mock.fetch_approvals().approvals_required.__sub__()' id='140291451716896'> more)") = <ExceptionInfo CannotMerge("Insufficient approvals (need <MagicMock name='mock.fetch_approvals().approvals_required.__sub__()' id='140291451716896'> more)") tblen=2>.value
```